runners/daily.rb

```diff
@@ -23,4 +23,24 @@
 
 CarrierWave.clean_cached_files!
 
+
+def delete_file_or_directory path
+  if File.directory? path
+    FileUtils.rm_rf path
+  else
+    FileUtils.rm path
+  end
```

Comment:
```suggestion
  File.directory? path ? FileUtils.rm_rf path : FileUtils.rm path
```
Quizás haga falta algún paréntesis acá... no lo probé.

Reply:
Si, faltan paréntesis